.flake8

```diff
@@ -0,0 +1,15 @@
+[flake8]
+# https://flake8.pycqa.org/en/2.5.5/warnings.html#error-codes
```

Comment:
A more update link is : 
https://flake8.pycqa.org/en/4.0.1/user/error-codes.html
https://pep8.readthedocs.io/en/latest/intro.html#error-codes

Is there a complete list of C9** errors?

Reply:
See https://www.flake8rules.com/.
I just removed the link from the file, because google search like "flake8 W503" easily gives the information.